components/brave_wallet_ui/components/desktop/views/portfolio/portfolio-asset.tsx

```diff
@@ -340,6 +342,14 @@ export const PortfolioAsset = (props: Props) => {
 
   const isNftAsset = selectedAssetFromParams?.isErc721
 
+  const isBuySupported = React.useMemo(() => {
```

Comment:
We should move `isBuySupported` and `isDepositSupported` to `useMultiChainBuyAssets` hook for better reuses